### PR TITLE
test: cover utils/check with tests

### PR DESCRIPTION
**Pull request by sdepold**
Adding some more tests for the utils' check file. Inspired by #13992 .

### CODE REVIEW

File: test/unit/utils/check.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const { expect } = require('chai');
+const { isColString } = require('sequelize/lib/utils/check');
```
Comment (ephys) [inline test/unit/utils/check.test.ts:2]: We could use `import` here since it's a typescript file

**Review by ephys — COMMENTED** [review]
I think we could add a test for `table.col` which should result in false
Otherwise, LGTM!

File: test/unit/utils/check.test.ts (hunk 1 above)
Comment (sdepold) [reply test/unit/utils/check.test.ts:2]: I tried and it didn't work. let me check

**Comment by sdepold** [comment]
I'm also adding more tests for the rest of the file right now

File: test/unit/utils/check.test.ts (hunk 1 above)
Comment (sdepold) [reply test/unit/utils/check.test.ts:2]: like this?

```import { expect } from "chai";
import { Utils } from "sequelize";

const { isPrimitive, isColString, canTreatArrayAsAnd, defaultValueSchemable } =
  Utils;
```

Comment (ephys) [reply test/unit/utils/check.test.ts:2]: Should work, but I expected this to work too:
```
import { isColString } from 'sequelize/lib/utils/check';
```

Comment (sdepold) [reply test/unit/utils/check.test.ts]: done

Comment (sdepold) [reply test/unit/utils/check.test.ts]: aha! somehow didn't work before but here we go

**Pull request merged**